packages/ws/src/ws/WebSocketManager.ts

```diff
@@ -271,22 +267,9 @@ export class WebSocketManager extends AsyncEventEmitter<ManagerShardEventsMap> i
 	}
 
 	public constructor(options: CreateWebSocketManagerOptions) {
-		if (!options.rest && !options.fetchGatewayInformation) {
```

Comment:
Shouldn't this still error if no fetchGatewayInformation function was passed? Considering the types only guarantee this for ts users, not runtime.